### DISCUSSION TITLE
Resizable component in the roadmap?

### DISCUSSION THREAD

**Discussion by TangkasOka** [huntabyte/shadcn-svelte#676] [Ideas]
Hi,

In the Dec 2023 release, Shadcn added a "resizable" component.  This is a resizable split pane.

https://ui.shadcn.com/docs/changelog#resizable

Is there a plan to add this to shadcn-svelte?

I'm in the proof-of-concept stage of migrating a web UI from ExtJS to Svelte with Shadcn, and I'm wondering whether a resizable will be part of shadcn-svelte in the next few months or if I should implement something myself (or use a non-shadcn-svelte resizable panel).

Thanks, Charlie

**Comment by shyakadavis** [comment]
Hey,

Hunter is currently working on a svelte port of the [react-resizable-panels](https://github.com/bvaughn/react-resizable-panels) (last I saw). So yeah, I'd guess we can expect it soon-ish.

I don't remember from which livestream, but he did show us a sneak peak.

**Reply by TangkasOka** [reply]
Many thanks!
